src/blocks/media-block/image-block/SimpleImageBlock.test.tsx

```
@@ -0,0 +1,342 @@
+    fireEvent.load(img);
+    
+    // Check if onLoad was called
+    expect(handleLoad).toHaveBeenCalled();
+  });
+  
+  it('shows error state when image fails to load', async () => {
+    // Mock the Image constructor to immediately trigger error
+    const originalImage = window.Image;
+    
+    // Create a mock implementation of Image that triggers an error
+    const mockImageImpl = function(this: HTMLImageElement | void): HTMLImageElement {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)